docs/JSON-RPC.md

```diff
@@ -1,14 +1,14 @@
 
-# Jamulus JSON-RPC Documentation
+# Jamulus JSON-RPC Server Documentation
 
 <!--
 This file is automatically generated from the source code.
 Do not edit this file manually.
 See `tools/generate_json_rpc_docs.py` for details.
 -->
 
-A JSON-RPC interface is available for both Jamulus client and server to allow programmatic access.
-To use the JSON-RPC interface, run Jamulus with the `--jsonrpcport <port> --jsonrpcsecretfile /file/with/a/secret.txt` options.
+A JSON-RPC server can be added to both Jamulus client and Jamulus server to allow programmatic access.
```

Comment:
Style: "Jamulus client" and "Jamulus server" should be "Jamulus Client" and "Jamulus Server" to comply with our terminology guide.